### PR TITLE
Swift 3 adaptions (⚠️NOT READY⚠️)

### PR DESCRIPTION
**Pull request by tiferrei**
Hello there, 

I really need to use this framework with Swift 3 so I tried to convert it, the framework itself runs and compiles perfectly, however when using it in my app it crashes when I try to invoque it, similar to issue #49.

Here's the code just in case it servers of any help.

Thank you,
tiferrei

### CODE REVIEW

**Comment by alissonbarauna** [comment]
Hey, 

I fixed this crash changing the line 190 to the shown on the image.

c1bf5a10-7a07-11e6-9ed7-016be03bd493

Well, I know it's not that applicable, because this line would receive the NSFontAttributeName automatically from the TextView.
But for me it's working fine now.

**Comment by tiferrei** [comment]
Hey @alissonbarauna, thank you so much, but my code is still crashing, I must have gone wrong somewhere else, do you mind if I take a look at your version of the code?

Thank you so much!
tiferrei

**Comment by alissonbarauna** [comment]
Tiago,

My JSAlert.swift is attached. I have made some changes on fonts and colors. But the rest is about the same.
If you have any question, please let me know.

Thanks.

[JSSAlert.swift.zip](https://github.com/openstakes/JSSAlertView/files/483169/JSSAlert.swift.zip)

**Comment by tiferrei** [comment]
That's perfect @alissonbarauna, thank you so much!

File: Pod/Classes/JSSAlertView.swift [hunk 1]
```diff
@@ -190,10 +190,10 @@ public class JSSAlertView: UIViewController {
 		// position text
 		if self.textView != nil {
 			let textString = textView.text! as NSString
-			let textAttr = [NSFontAttributeName:textView.font as! AnyObject]
```
Comment (alissonbarauna) [inline Pod/Classes/JSSAlertView.swift]: @tiferrei as mentioned, the problem seems to be on this line! I'm not advanced on Swift and I couldn't figure out whats's wrong. Setting textAttr to static attributes fixed the problem.
Tks

**Pull request closed without merging**